apps/ledger-live-mobile/src/components/Nft/NftListItem.tsx

```diff
@@ -188,9 +232,37 @@ const NftMediaComponent = ({ status, metadata, nftAmount }: NftMediaProps) => {
         >
           {t("wallet.nftGallery.media.tag", { count: nftAmount.toNumber() })}
         </Tag>
+
+        {selectable && (
+          <Flex position="absolute" bottom={"20px"} left={"10px"}>
+            <SelectNft isSelected={isSelected} />
+          </Flex>
+        )}
+      </Box>
+    );
+  }
+
+  if (selectable) {
+    return (
+      <Box position="relative">
+        <NftMedia
+          style={[
+            styles.image,
+            {
+              opacity: isSelected ? 0.3 : selectable ? 0.7 : 1,
```

Comment:
is it expected that the opacity values are different from those used in the previous if block?

IMO this entire component could be simplified/refactored slightly by returning 1 block JSX with the conditions inside. It would ensure consistency for the common props & style

Something like that:
```
const NftMediaComponent = () => {
  return (
    <Box>
      <NftMedia style={yourStyleLogicDependingOnSelectableEtc} otherPropsHere />
      {moreThanOne ? <TheTagThatDisplaysTheCount /> : null}
      {selectable ? <SelectableCheckboxIndicator isSelected={isSelected}> : null}
    </Box>
  )
}
```